docs/pages/management/operations/enroll-agent-into-automatic-updates.mdx

```diff
@@ -34,15 +34,15 @@ updates.
 ## Requirements
 
 <Tabs>
-<TabItem label="Self-hosted">
+<TabItem label="Self-hosted" scope={"enterprise"}>
 - A Teleport Enterprise agent, either:
   - started via systemd on a distribution using the `apt` or `yum` package managers
   - deployed with the `teleport-kube-agent` Helm chart
 - automatic update infrastructure set up. For Self-Hosted users this means you
   already followed [this guide](./self-hosted-automatic-agent-updates.mdx) and
   know your version server URL and release channel
 </TabItem>
-<TabItem label="Teleport Cloud">
+<TabItem label="Teleport Cloud"  scope={["cloud", "team"]}>
```

Comment:
This page is only configured for the `enterprise` and `cloud` scopes. Should we add `team` to its `forScopes`?

Reply:
@ptgott added to scopes